### PR TITLE
Add doc for lod_rank_table.

### LINKED ISSUE #7024: Add doc for lod_rank_table

### PR DESCRIPTION
**Pull request by pkuyym**
Resolves #7024

### CODE REVIEW

File: python/paddle/v2/fluid/layers/control_flow.py [hunk 1]
```diff
@@ -397,9 +397,40 @@ def complete(self):
 
 
 def lod_rank_table(x, level=0):
-    """
-    This function creates an operator for creating a LOD_RANK_TABLE
-    using the input x.
+    """LoD Rank Table Operator. Given an input variable `x` and a LoD level,
```
Comment (JiayiFeng) [inline python/paddle/v2/fluid/layers/control_flow.py]: `LoD level` ==> `number of Lod level`

File: python/paddle/v2/fluid/layers/control_flow.py [hunk 2]
```diff
@@ -397,9 +397,40 @@ def complete(self):
...
+    this layer creates a LodRankTable object. A LoDRankTable object contains a
+    list of bi-element tuples and each tuple consists of an index and a length.
```
Comment (JiayiFeng) [inline python/paddle/v2/fluid/layers/control_flow.py]: > each tuple consists of an index and a length, **both of them are int**.

File: python/paddle/v2/fluid/layers/control_flow.py [hunk 3]
```diff
@@ -397,9 +397,40 @@ def complete(self):
...
+    For given level's LoD information, the index is the sequence position and
```
Comment (JiayiFeng) [inline python/paddle/v2/fluid/layers/control_flow.py]: It's not easy to understand what is `level's LoD information`.

File: python/paddle/v2/fluid/layers/control_flow.py (hunk 1 above)
Comment (pkuyym) [reply python/paddle/v2/fluid/layers/control_flow.py]: Done.

File: python/paddle/v2/fluid/layers/control_flow.py (hunk 2 above)
Comment (pkuyym) [reply python/paddle/v2/fluid/layers/control_flow.py]: Done.

File: python/paddle/v2/fluid/layers/control_flow.py (hunk 3 above)
Comment (pkuyym) [reply python/paddle/v2/fluid/layers/control_flow.py]: Done.

File: python/paddle/v2/fluid/layers/control_flow.py [hunk 4]
```diff
@@ -397,9 +397,49 @@ def complete(self):
...
+                Create lod rank table:
+                    lod_rank_table_obj = lod_rank_table(x, level=1)
+
+                Get:
+                    lod_rank_table_obj.items() = [(0, 5), (1, 1), (2, 1)]
+
+    Args:
+        x (Variable): Input variable, a LoDTensor based which to create the lod
+            rank table.
+        level (int): Specify the LoD level.
```
Comment (JiayiFeng) [inline python/paddle/v2/fluid/layers/control_flow.py]: Specify on which LoD level create the LoDRankTable object.

**Pull request merged**